rct229/rulesets/ashrae9012019/section10/section10rule14.py

```diff
@@ -170,7 +170,7 @@ def get_calc_vals(self, context, data=None):
                     )
 
             if total_capacity_b is not None:
-                total_capacity_b = total_capacity_b.magnitude
+                total_capacity_b = total_capacity_b.to("Btu/h").magnitude
```

Comment:
@weilixu Do you agree with this update?

Reply:
I see, the HEATPUMP_CAPACITY_LOW_THRESHOLD etc does not have unit assignment. We typically want to keep those constants as quantity objects (see example; https://github.com/pnnl/ruleset-checking-tool/blob/master/rct229/rulesets/ashrae9012019/section19/section19rule13.py#L46-L47)

The pint internally handles unit conversion when comparing for example "w" and "btu/h". So if you add units to these constants, then you do not need to extract mag from the quantity object.

Reply:
I used the magnitude because of a previous decision to keep the lookup tables using integers only